### PR TITLE
Add type file with `getMulti` 

### PR DESCRIPTION
**Pull request by justjake**
I don't like the workflow of using patch-package on @types/memjs, it's hard to iterate on that.

### CODE REVIEW

File: lib/memjs/memjs.d.ts [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+   * The callback signature is:
+   *
+   *     callback(err, values, flags)
+   *
+   * @param keys
+   * @param callback
+   */
+  getMulti<Keys extends string>(
+    keys: Keys[]
+  ): Promise<{ values: { [K in Keys]?: Buffer | null }; flags: Buffer | null }>;
```
Comment (blackmad) [inline lib/memjs/memjs.d.ts:169]: you're sure? these really seem like strings to me based on the unittests, I think the docs in memjs had been wrong?

**Review by blackmad — CHANGES_REQUESTED** [review]

File: lib/memjs/memjs.d.ts (hunk 1 above)
Comment (blackmad) [reply lib/memjs/memjs.d.ts:169]:
```
  var assertor = function(err, val, flags) {
    console.log('types')
    console.error(val['hello1'])
    console.error(typeof val['hello1'])
    t.deepEqual({
      hello1: 'world1',
      hello3: 'world3',
      hello4: 'world4',
    }, val);
    t.false(flags);
    t.equal(null, err);
  };
  client.getMulti(['hello1', 'hello2', 'hello3', 'hello4'], assertor);
  testAllCallbacksEmpty(t, dummyServer1);
  testAllCallbacksEmpty(t, dummyServer2);

  return client.getMulti(['hello1', 'hello2', 'hello3', 'hello4']).then(function(res) {
    assertor(null, res.values, res.flags);
```

```
world1
string
world1
string
```

Comment (justjake) [reply lib/memjs/memjs.d.ts:169]: Very weird. Here's me, in a nodejs REPL:
image

Test script using promise:

image

**Comment by blackmad** [comment]
hmmm okay I want to figure this out today, I was surprised the docs were
"wrong" ... will look a bit more, is confusing me.

**Comment by justjake** [comment]
image

**Review by blackmad — APPROVED** [review]

**Pull request merged**